pkgs/objective_c/ffigen_objc.yaml

```diff
@@ -70,6 +70,7 @@ objc-categories:
   include:
     - NSDataCreation
     - NSExtendedMutableData
+    - NSNumberCreation
```

Comment:
Isn't there a table that you have to update in `ffigen`?

Reply:
Not for categories. That table prevents users from defining classes etc that conflict with the ones in package:objective_c, but if they re-define these extension methods that doesn't actually cause any problems.